config/jobs/kubernetes-sigs/special-resource-operator/special-resource-operator-presubmits.yaml

```diff
@@ -45,9 +9,9 @@ presubmits:
       description: "lint the source code of special-resource-operator."
     spec:
       containers:
-      - image: golang:1.17
+      - image: golang:1.18
         command:
-        - scripts/test-infra/lint.sh
+        - ci/prow/lint.sh
```

Comment:
WDYT about removing the extension? The [Google style guide](https://google.github.io/styleguide/shellguide.html#file-extensions) strongly recommends no extension for executables.

Reply:
I do like the `.sh` extension. It lets you know that this is a bash script and not just a compiled binary.
Also, no crucial, but this also aligned with what we will soon have with NFD - https://github.com/kubernetes-sigs/node-feature-discovery/tree/master/scripts/test-infra

Reply:
Fixed.